packages/react/src/primitives/types/checkbox.ts

```diff
@@ -0,0 +1,18 @@
+import { FlexProps } from './flex';
+import { InputProps } from './input';
+
+export interface CheckboxProps extends FlexProps, InputProps {
+  /**
+   * The label text
+   */
+  children: string;
```

Comment:
I think this should match the other field [label](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/types/field.ts#L22) type:
```suggestion
  children: FieldProps['label'];
```